packages/gatsby-theme-newrelic/src/components/Callout.js

```diff
@@ -11,16 +11,34 @@ const VARIANTS = {
 
 const styles = {
   variant: {
+    [`${VARIANTS.CAUTION}-title`]: css`
+      color: var(--color-red-400);
+    `,
     [VARIANTS.CAUTION]: css`
-      border-left: 4px solid var(--color-red-400);
+      border-top: 1px solid var(--color-red-400);
+      border-right: 1px solid var(--color-red-400);
+      border-bottom: 1px solid var(--color-red-400);
+      border-left: 6px solid var(--color-red-400);
```

Comment:
I think there is an opportunity for consolidation here. Rather than listing all 4 of these as separate properties, I'd use an override for the thicker border. This would conslidate the styles to:

```
border: 1px solid var(--color-red-400);
border-left-width: 6px;
```

Use this technique for the other variants as well.